coserv/api/handler.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package api
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/fxamacker/cbor/v2"
+	"github.com/gin-gonic/gin"
+	"github.com/lestrrat-go/jwx/v2/jwk"
+	"github.com/veraison/corim/coserv"
+	"github.com/veraison/services/config"
+	"github.com/veraison/services/coserv/endorsementdistributor"
+	"github.com/veraison/services/log"
+	"github.com/veraison/services/proto"
+	"go.uber.org/zap"
+)
+
+var (
+	tenantID  = "0"
+	CoservMTs = []string{
+		"application/coserv+cbor",
+		"application/coserv+cose",
+	}
+)
+
+type Handler struct {
+	Logger                *zap.SugaredLogger
+	EndorsementDistibutor endorsementdistributor.IEndorsementDistributor
+}
+
+func NewHandler(endorsementdistributor endorsementdistributor.IEndorsementDistributor, logger *zap.SugaredLogger) Handler {
```

Comment:
```suggestion
func NewHandler(
	endorsementdistributor endorsementdistributor.IEndorsementDistributor,
	logger *zap.SugaredLogger,
) Handler {
```

Reply:
Addressed in [7bcdc10](https://github.com/veraison/services/pull/360/commits/7bcdc105cc1bd48a395777dc87d10bcfef476e18)